src/main/java/org/sonarsource/auth/github/GitHubSettings.java

```diff
@@ -36,6 +36,7 @@
 @ServerSide
 public class GitHubSettings {
 
+  //TODO remove TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.auth.github%3Asonar-auth-github-plugin&pullRequest=27&issues=AWJKrtsa5ZbnBykCSSP2&open=AWJKrtsa5ZbnBykCSSP2)

[ISSUE_KEY]: AWJKrtsa5ZbnBykCSSP2